packages/edit-site/src/hooks/use-theme-style-variations/use-theme-style-variations-by-property.js

```diff
@@ -160,8 +160,12 @@ export default function useThemeStyleVariationsByProperty( {
 					cloneDeep( variation ),
 					property
 				);
+
 				// Remove variations that are empty once the property is filtered out.
-				if ( Object.keys( variationFilteredByProperty ).length === 0 ) {
+				if (
+					variation.title !== 'Default' &&
```

Comment:
If I'm not wrong, this is a translated string so it might break. We could test `__( 'Default' )` instead but I wonder if there's a better way to identify this (a flag or something)

Reply:
Good catch. We do this elsewhere too: https://github.com/WordPress/gutenberg/blob/79d79ff565daa0a88746eb85bccd2a0778ac7a93/packages/edit-site/src/components/global-styles/hooks.js#L126

Reply:
Done in 2cfcf6ba4e